source/administration/changelog.md

```diff
@@ -6,10 +6,10 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.26 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
-- **v5.26.1, release day TBD**
-  - Fixing an issue where the [LDAP PictureAttribute feature](https://docs.mattermost.com/administration/config-settings.html#profile-picture-attribute) only works with users signing in with LDAP but not with SAML authentication. [MM-27852](https://mattermost.atlassian.net/browse/MM-27852)
-  - Fixing an issue where the mobile app sessions are getting logged out unexpectedly when ``ExtendSessionLengthWithActivity`` is enabled. [MM-27184](https://mattermost.atlassian.net/browse/MM-27184)
-  - Fixing an issue where null AuthData value causes kernel panic during LDAP sync. [MM-27965](https://mattermost.atlassian.net/browse/MM-27965)
+- **v5.26.1, released 2020-08-26**
```

Comment:
Should this say the 25th instead of the 26th? Or with time zones and business hours do we mark it as the 26th?